### PR TITLE
NTP-863: Change log level from warning to info when no Tuition Partner is found

### PR DESCRIPTION
**Pull request by hasan3d**
## Context

Currently, when someone passes an invalid tuition partner id not found on our system, we log a warning that creates an alert for us to look into that. As the tuition partner id is not found, an error classed as 404 error, we will log that as info instead of a warning that will not trigger an alert.

## Changes proposed in this pull request

Change log level from warning to information when there is no TP found.

## Guidance to review

Ensure that when no Tuition Partner is found for a given id on the`TuitionPartner.cshtml.cs`, the error is logged as information, and we do not get an alert for that error.

## Link to Jira ticket

[https://dfedigital.atlassian.net/browse/NTP-863](https://dfedigital.atlassian.net/browse/NTP-863)

## Things to check

- [x] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [x] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [x] `dotnet format` has been run in the repository root
- [ ] Test coverage of new code is at least 80%
- [ ] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [ ] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [x] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

File: UI/Pages/TuitionPartner.cshtml.cs [hunk 1]
```diff
@@ -79,9 +79,9 @@ public async Task<IActionResult> OnPostRemoveFromShortlist([FromBody] string seo
     private ArgumentException GetArgumentException(string name) => new($"{name} is null or whitespace");
     private JsonResult GetShortlistJsonResult(bool status) => new(new { Updated = status });
 
-    private IActionResult ReturnNotFound(string logMessage)
+    private IActionResult ReturnNotFoundWithLogging(string logMessage, LogLevel logLevel)
     {
-        _logger.LogWarning("{LogMessage}", logMessage);
+        _logger.Log(logLevel, "{LogMessage}", logMessage);
```
Comment (ashward) [inline UI/Pages/TuitionPartner.cshtml.cs:84]: Not sure we need to pass in the log level, as it's only ever passed in as `LogLevel.Information`, so you could have just updated this line to
```suggestion
        _logger.LogInformation("{LogMessage}", logMessage);
```

**Review by ashward — APPROVED** [review]
Other than the suggestion about it being slightly over-engineered (which you can take or leave!), it looks okay to me.

File: UI/Pages/TuitionPartner.cshtml.cs (hunk 1 above)
Comment (hasan3d) [reply UI/Pages/TuitionPartner.cshtml.cs:84]: If we make the method more flexible, we could reuse it for any other log level. That was my thought process. Happy to apply your suggested approach, as well.

Comment (ashward) [reply UI/Pages/TuitionPartner.cshtml.cs:84]: TBH this whole `ReturnNotFound` method is pretty superfluous anyway, I reckon!

Comment (hasan3d) [reply UI/Pages/TuitionPartner.cshtml.cs:84]: Yeah, agree.

**Pull request merged**